### ISSUE TITLE
[Serious] Text content lacks 4.5:1 contrast ratio

### ISSUE THREAD

**Issue by ntallman** [APTrust/registry#74]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87704 |
| **Checkpoint** | 1.4 Distinguishable › Color Contrast (regular text) (1.4.3.a) |
| **Impact** | Serious |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 03 Objects page |
| **Test Case Name** | AP Trust Registry (Demo) |
| **URL** | <https://demo.aptrust.org/objects?state=A> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Product** | Website |
| **Date Reported** | 05/11/26 at 8:57 am |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | a3caccess+@gmail.com |

## Description

The contrast ratio between text and its background is not at least 4.5:1.
Pattern. In Firefox, when you expand More Filters, then select one of the calendar select buttons, the days Mon - Fri in the calendar are light gray which has a low color contrast and is difficult to see. See Work Items User Experience Video at 17:45.Also occurs on Pages 04 Work Items (More Filters expanded) and 06 Alerts.

## Recommendation

Rule : 
Small text (under 18 point regular font or 14 point bold font) MUST have a contrast ratio of at least 4.5 to 1 with the background.

How To Fix: 
Fix this issue by adjusting the text and/or background to increase the contrast to at least 4.5 to 1.

Background: 
People who have low vision or are colorblind may have difficulty reading text if the contrast between the text its background is insufficient. When the contrast ratio between text and its background is adequate, people who have low vision or are colorblind are more likely to be able to read the text.

## Source Code

```
N/A
```

## Screenshots

![04_color contrast calendar 1](https://raw.githubusercontent.com/APTrust/registry/master/docs/accessibility-audit/screenshots/87704/04_color_contrast_calendar_1.png)
